android/history.md

```diff
@@ -3,7 +3,8 @@
 
 ## 11.0 alpha
 * Move to 11.0
-* Add round launcher icons
+* Add round launcher icons (#1077)
+* Breaking change - Remove ad-hoc distribution of keyboards via `keyman://` protocol (#1109)
```

Comment:
Suggest we don't use phrase "Breaking change" because while it is a breaking change, it's more of a removal of old infrastructure. "Breaking change" sounds scary but this isn't really scary -- vast majority of people won't even notice. Instead, maybe `Remove deprecated ad-hoc distribution of keyboards via `keyman://` protocol (#1109)`